### ISSUE TITLE
go-import meta tag isn't defined for the examples submodule

### ISSUE THREAD

**Issue by henryjcee** [go-gorm/gen#1226]
I have contributed to [this issue](https://github.com/go-gorm/playground/issues/751) in the playground repo and as discussed, the core issue appears to be that `/gen/examples` package has been made a submodule. This breaks the existing playground import structure. 

I'm not an expert on the Go module system but I suspect that this is because `go mod` is searching `https://gorm.io/gen/examples` for a meta tag that should look something like 

```
<meta name="go-import" content="gorm.io/gen/examples git https://github.com/go-gorm/gen/examples">
```

This url currently 404s and as a result the import:

```
gorm.io/gen/examples v0.3.26
```

fails to resolve and prevents running the playground. The playground can be run by checking out the `gen` project and using a replace directive but right now I _think_ this is right solution.

**Comment by tr1v3r** [comment]
The example is just a usage demonstration of the project and is not intended to be used as a Go SDK for import.

**Comment by henryjcee** [comment]
Indeed, but it is imported [here](https://github.com/go-gorm/playground/blob/4f84925d5e87c3c8b9ac5cfc14a7db47fb0ea4f7/gen.go#L5) in playground.
